machine-learning/ml_tips/README.md

```diff
@@ -0,0 +1,74 @@
+# ML tips workflow
+
+This workflow aims to explain common ML techniques using Hivemall and Treasure Workflow.
+
+Using [titanic data](https://github.com/amueller/scipy-2017-sklearn/blob/master/notebooks/datasets/titanic3.csv), this workflow explains the following basic techniques.
+
+- Random/stratified splitting for train and test data
+- Feature normalization
+- Missing value imputation
+
+## Input
+
+This workflow assumes a table as follows:
+
+|pclass<br/>`int`|survived<br/>`int`|name<br/>`string`|sex<br/>`string`|age<br/>`int`|sibsp<br/>`int`|parch<br/>`int`|ticket<br/>`string`|fare<br/>`double`|cabin<br/>`string`|embarked<br/>`string`|boat<br/>`string`|body<br/>`int`|home.dest<br/>`string`|
+|---------|------|---------|--------|-------|-------|-------|--------|-------|-------|-----------|--------|---------|-------|
+|1|1|"Allen, Miss. Elisabeth Walton"|female|29|0|0|24160|211.3375|B5|S|2||"St Louis, MO"|
+|1|1|"Allison, Master. Hudson Trevor"|male|0.9167|1|2|113781|151.5500|C22 C26|S|11||"Montreal, PQ / Chesterville, ON"|
+|1|0|"Allison, Miss. Helen Loraine"|female|2|1|2|113781|151.5500|C22 C26|S|||"Montreal, PQ / Chesterville, ON"|
+|...|...|...|...|...|...|...|...|...|...|...|...|...|...|
+
+Here is a description of each column.
```

Comment:
In this context, we'd better to use "a" instead of "the" since this is not the only description. ref:  https://forum.wordreference.com/threads/here-is-a-the-description-of.2978732/